src/Suave/Http.fs

```diff
@@ -220,7 +220,7 @@ module Http =
      
 
     let not_modified : WebPart =
-      fun ctx -> { ctx with response = {status = HTTP_304; headers = []; content = Bytes [||] }} |> succeed
+      fun ctx -> { ctx with response = {status = HTTP_304; headers = []; content = Bytes [||]; writePreamble = true }} |> succeed
```

Comment:
Can we encode it as a field of SocketTask? Are there any other than SocketTask that need to avoid writing preambles?